<launch>
    <arg name="urdf" default="" />
    <arg name="interface" default="can0" />
    <arg name="end_mode" default="newteacher" />
    <arg name="joy_dev" default="/dev/input/js0" />
    <node name="airbot_arm_ros" pkg="ros_interface" type="airbot_arm_ros" output="screen">
        <param name="urdf" value="$(arg urdf)" />
        <param name="interface" value="$(arg interface)" />
        <param name="end_mode" value="$(arg end_mode)" />
    </node>
    <node name="joy_proxy" pkg="ros_interface" type="joy_proxy" output="screen" />
    <node name="joy" pkg="joy" type="joy_node" output="screen" >
        <param name="dev" value="$(arg joy_dev)" />
    </node>
</launch>
